code/__DEFINES/tools.dm

```diff
@@ -21,6 +21,10 @@
 #define TOOL_ROLLINGPIN "rolling pin"
 /// Can be used to scrape rust off an any atom; which will result in the Rust Component being qdel'd
 #define TOOL_RUSTSCRAPER "rustscraper"
+/// Currently used only in borg botany omnitool, you may add some funny interaction with it later
+#define TOOL_CULTIVATOR "cultivator"
+#define TOOL_SECATEUR "secateur"
+#define TOOL_HATCHET "hatchet"
```

Comment:
This is really bad, you shouldn't be adding tool types purely for the sake of making poorly written code (omnitools) work